src/reynir_correct/pattern.py

```diff
@@ -1816,14 +1816,29 @@ def né(self, match: SimpleTree) -> None:
         if c is None:
             return
         start, end = c.span[0], c.span[1]
+        if (
+            match.root.first_match("'hvorki'") is not None
+            or "eitt né neitt" in match.root.text
+        ):
+            # Check to see if VillaNé (P_NT_Né) has been activated
+            # and should be deleted
+            for ann in self._ann:
+                if ann.code == "P_NT_Né" and ann.start == start and ann.end == end:
+                    self._ann.remove(ann)
+            return
+
+        for ann in self._ann:
```

Comment:
Maybe a check of this kind belongs at a higher level, i.e. generally checking that the same annotation is not repeated, and if it is, remove it

Reply:
See comment above; not exactly the same annotation.